readme.md

```diff
@@ -65,6 +66,7 @@ $ np --help
     --no-tests          Skips tests
     --yolo              Skips cleanup and testing
     --no-publish        Skips publishing
+    --preview           Only show what np is going to execute without running anything
```

Comment:
The description is a bit wordy. Can you make it more succinct? Also no need to mention `np`.